components/haptics/index.js

```diff
@@ -41,6 +40,10 @@ AFRAME.registerComponent('haptics', {
       this.addEventListeners();
     } else {
       this.el.addEventListener('controllerconnected', function init () {
+        // If no tracked-controls yet exists, add it here
```

Comment:
I believe that if  `controllerconnected` fires tracked-controls should be defined.

Reply:
added that assumption in